src/averageAges.js

```diff
@@ -38,6 +50,16 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const femalesAverageAge = withChildren
+    ? people.filter((person) => person.sex === 'f').filter(
+      (person) => people.filter((mum) => mum.mother !== null)
```

Comment:
do you need this check to be so complex? try to do it without comparing to !== null